src/WebJobs.Script/Description/ScriptType.cs

```diff
@@ -15,6 +15,7 @@ public enum ScriptType
         FSharp,
         DotNetAssembly,
         TypeScript,
-        Unknown
+        Unknown,
+        Proxy
```

Comment:
This doesn't belong in this enum - all the other enum items are about the language type.

Rather than this, I'd be ok with you adding an IsProxy property to FunctionMetadata until we come up with something better.